resources/assets/js/videos/components/videoScreen.vue

```diff
@@ -11,18 +11,17 @@
             :position="mousePosition"
             ></label-tooltip>
         <div class="controls">
-            <div class="btn-group"
-                v-if="showPrevNext">
-                 <control-button
-                    icon="fa-chevron-left"
-                    title="Previous video"
+            <div class="btn-group">
+                <control-button
+                    v-if="showPrevNext"
+                    icon="fa-step-backward"
+                    title="Previous video 𝗟𝗲𝗳𝘁 𝗮𝗿𝗿𝗼𝘄"
                     @click="emitPrevious"
                     ></control-button>
-            </div>     
-            <div class="btn-group">
                 <control-button
+                    v-if="!seeking && jumpStep!=0"
```

Comment:
You should disable the button instead of hiding it while `seeking`, otherwise the button toolbar jumps around each time the feature is used:

```suggestion
                    v-if="jumpStep!=0"
                    :disabled="seeking"
```